addons/website_sale_loyalty/views/website_sale_templates.xml

```diff
@@ -43,6 +43,9 @@
                                                     <t t-set="program" t-value="reward.program_id"/>
                                                     <t t-set="points" t-value="coupon._format_points(website_sale_order._get_real_points_for_coupon(coupon))"/>
                                                     <t t-if="program.program_type not in ['ewallet', 'promo_code', 'loyalty'] and (program.trigger == 'with_code' or (program.trigger == 'auto' and program.applies_on == 'future'))">
+                                                        <t t-if="program.trigger == 'auto' and program.applies_on == 'future'">
```

Comment:
`program.trigger == 'auto'` should always `program.applies_on == 'future'` thnx to [above condition](https://github.com/odoo/odoo/pull/164868/files#diff-1b1188f3b08036ead004d84e46692a203d9602fc3c7fdbf2d91b9d6c0c423220R45)
Also you can right all this condition on `<t>` tag instead writing half on `<strong>` tag